mars-mission/src/Components/forms.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { countriesList } from "./countriesList/countries";
+import { diets } from "./Dietary/diets";
+import "./css/forms.css";
+
+class Form extends React.Component {
+  state = {
+    fullName: "",
+    date: {},
+    countries: {},
+    diet: {},
+    bodyResponse: "",
+    formCompleted: false
+  };
+
+  handleChange = e => {
+    // debugger;
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  populateDiets = () => {
+    return diets.map(diet => {
+      return <option value={diet}>{diet}</option>;
+    });
+  };
+
+  populateCountries = () => {
+    return countriesList.map(countries => {
+      return <option value={countries.code}>{countries.name}</option>;
+    });
+  };
+
+  handleSubmit = () => {
+    const { formCompleted } = this.state;
+    if (formCompleted === false) {
+      this.setState({
+        formCompleted: true
+      });
+    }
+    // debugger;
+  };
+
+  completedForm = () => {
+    const { fullName, date, countries, diet, bodyResponse } = this.state;
+    debugger;
+    return (
+      <div>
+        <p>Form Completed</p>
+        <h3>{fullName}</h3>
+
+        <ul>
+          <li>{date}</li>
+          <li>{countries}</li>
+          <li>{diet}</li>
+          <li>{bodyResponse}</li>
+        </ul>
+      </div>
+    );
+  };
+
+  handleForm = e => {
+    // debugger;
+    e.preventDefault();
+    let {
```

Comment:
try using const instead. Always use const unless you're going to reassign the value.